launcher/launcher.cpp

```diff
@@ -196,12 +196,16 @@ void load_file_list(std::string directory) {
       continue;
     }
 
-    if(ext == "bmp" || ext == "spritepk" || ext == "spriterw") {
+    // spritepk and spriterw are old, transitional long exts, sspk and ssrw are the file-association compatible short names
+    if(ext == "bmp" || ext == "spritepk" || ext == "spriterw" || ext == "sspk" || ext == "ssrw") {
```

Comment:
I don't think spritepk was even referenced anywhere until now... and spriterw is only a few days old, don't think much would break if you just dropped those.

Reply:
True, I'm forging new ground actually using these features :laughing:

Reply:
Tbf I'm not entirely keen on "sspk" and "ssrw" but it's something. It should just be "ss32" or something, and the format is described in the file. Then `.save()` can take `packed` and `rle` args for when we're feeling really bored and want to implement them.

Reply:
Yeah, all the info is already in the header for loading... though putting packing code in `save` would end up including it in the firmware... (Solve that one later I guess)